requirements.txt

```diff
@@ -1,7 +1,7 @@
 keras2onnx
 numpy
 onnx
-onnxconverter-common>=1.6.5, <1.7.0
```

Comment:
Don't change that, this is mandatory, according to LT team's comments.

Reply:
Fixed. I don't remember the explicit Reason though.